apps/src/code-studio/pd/foorm/FoormEditor.jsx

```diff
@@ -87,6 +87,7 @@ class FoormEditor extends React.Component {
 
   componentDidMount() {
     this.props.populateCodeMirror();
+    this.previewFoorm();
```

Comment:
added this here because I reordered some of the setup steps due to refactoring in FoormEditorManager, and found the editor relies on seeing a props change to do the initial preview. This change ensures if the props are set before the editor loads it will still preview the form.